src/Uno.UI/UI/Xaml/Duration.cs

```diff
@@ -16,7 +16,7 @@ public Duration(TimeSpan timeSpan)
 		}
 
 		public DurationType Type;
-		public TimeSpan TimeSpan;
+		public TimeSpan TimeSpan { get; set; }
```

Comment:
Note: I think it's a property only on WinUI, but not on UWP. Because otherwise the sync generator should have caught it.